classes/game.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'item'
+require 'date'
+
+# Game class
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(date, multiplayer, last_played_at)
+    super(date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    super && Date.today - Date.parse(@last_played_at) > 1464
+  end
```

Comment:
Please consider returning true from the method if `@last_played_at` is older than `2 years` according to the project requirements. Currently, it's checking if the difference is greater than `4years`

Please consider making the method private as indicated in the [UML diagram](https://github.com/microverseinc/curriculum-ruby/raw/main/group-capstone/images/catalog_of_my_things.png) the method is marked private.